functions/__fzf_search_shell_variables.fish

```diff
@@ -5,12 +5,12 @@ function __fzf_search_shell_variables --argument-names set_show_output set_names
     # inform users who use custom key bindings of the backwards incompatible change
     if test -z "$set_names_output"
         set_color red
-        printf '\n%s\n' '__fzf_search_shell_variables now requires arguments so you have to update your key bindings.'
-        printf '%s\n\n' 'Please see github.com/PatrickF1/fzf.fish/releases/tag/v5.0 for the resolution.'
+        printf '\n%s\n' '__fzf_search_shell_variables now requires arguments so you have to update your key bindings.' >&2
+        printf '%s\n\n' 'Please see github.com/PatrickF1/fzf.fish/releases/tag/v5.0 for the resolution.' >&2
         set_color normal
 
         commandline --function repaint
-        return
+        return 22 # 22 means invalid argument in POSIX
```

Comment:
Even though probably no one else will ever check the error code, I had to modify this to make it more testable as before it was exiting with a status code of 0.